rclrs/src/subscription/readonly_loaned_message.rs

```diff
@@ -45,3 +45,9 @@ where
         }
     }
 }
+
+// SAFETY: The functions accessing this type, including drop(), shouldn't care about the thread
+// they are running in. Therefore, this type can be safely sent to another thread.
+unsafe impl<'a, T> Send for ReadOnlyLoanedMessage<'a, T> where T: RmwMessage {}
+// SAFETY: This type has no interior mutability, in fact it has no mutability at all.
```

Comment:
```suggestion
// SAFETY: This type is not mutable.
```

Reply:
Thanks for your suggestion – I like the previous version a tiny bit better though since "interior mutability" directly references the criterion for a type to be `Sync`. Since you already approved, I assume leaving it like that is also fine with you :)